src/generators/app.ts

```diff
@@ -455,69 +456,88 @@ class App extends Generator {
       )
       devDependencies.push(
         '@oclif/dev-cli@^1',
-        '@oclif/plugin-help@^2',
-        'globby@^8',
+        '@oclif/plugin-help@^3',
+        'globby@^11',
       )
       break
     case 'sfdx-plugin':
       dependencies.push(
         '@oclif/command@^1',
         '@oclif/config@^1',
         '@oclif/errors@^1',
-        '@salesforce/command@^2',
+        '@salesforce/command@^3',
         '@salesforce/core@^2'
       )
       devDependencies.push(
         '@oclif/dev-cli@^1',
-        '@oclif/plugin-help@^2',
-        'globby@^8',
-        '@salesforce/dev-config@1.4.1'
+        '@oclif/plugin-help@^3',
+        'globby@^11',
+        '@salesforce/dev-config@^2',
+        '@salesforce/dev-scripts@^0',
+        '@salesforce/prettier-config@^0',
+        '@salesforce/ts-sinon@^1',
+        '@types/jsforce@^1.9.29',
+        '@typescript-eslint/eslint-plugin@^4',
+        '@typescript-eslint/parser@^4',
+        'eslint-config-prettier@^8',
+        'eslint-config-salesforce@^0',
+        'eslint-config-salesforce-license@^0',
```

Comment:
plugin-generator is intended for customer use...let's not put our copyright on it by default

Reply:
@mshanemc I'll remove salesforce license, what about our config?